installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc

```diff
@@ -13,6 +13,12 @@ Use the following procedures to install an {product-title} cluster using the Age
 
 * You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
 * You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
```

Comment:
Would the above prereq still make sense if you removed this note and mentioned proxy there instead? 

Such as having the prereq say "If you use a firewall or proxy, you ..."

It's just that this note seems almost like another separate prereq and it might get missed by people skimming over this note for something about firewalls. Let me know what you think!

Reply:
Sure thing, @skopacz1.  I'll merge firewall and proxy into a single line.